CMakeLists.txt

```diff
@@ -3,6 +3,13 @@ project( SFCGAL )
 
 set( CMAKE_DEBUG_POSTFIX "d" )
 
+#
+# Cmake policies
+#
+if(POLICY CMP0077)
+    cmake_policy(SET CMP0077 NEW)
+endif()
+
```

Comment:
What is this cryptic CMP077 policy ? Looks like James Bond's permit to kill

Reply:
Yes, It' my gold fingers :sunglasses: :rofl: 

This [policy](https://cmake.org/cmake/help/git-stage/policy/CMP0077.html) is for options on BOOST static